pkgs/by-name/go/godini/package.nix

```diff
@@ -0,0 +1,49 @@
+{
+  buildGoModule,
+  fetchFromGitHub,
+  godini,
+  lib,
+  nix-update-script,
+  testers,
+}:
+
+buildGoModule rec {
+  pname = "godini";
+  version = "1.0.0";
+
+  src = fetchFromGitHub {
+    owner = "bilbilak";
+    repo = "${pname}";
+    rev = "v${version}";
+    hash = "sha256-83OAddIoJzAUXPZKGnAx8XPKrdSmtc1EIJUDmRHTU/U=";
+  };
+
+  vendorHash = "sha256-hocnLCzWN8srQcO3BMNkd2lt0m54Qe7sqAhUxVZlz1k=";
+
+  ldflags = [
+    "-s"
+    "-w"
+    "-X github.com/bilbilak/${pname}/config.Version=${version}"
+  ];
+
+  passthru = {
+    tests = {
+      version = testers.testVersion {
+        package = godini;
+        command = "${pname} --version";
+      };
+    };
+
+    updateScript = nix-update-script { };
+  };
+
+  meta = {
+    changelog = "https://github.com/bilbilak/${pname}/blob/main/CHANGELOG.md";
+    description = "INI Configuration Management Tool";
+    homepage = "https://github.com/bilbilak/${pname}";
+    license = lib.licenses.gpl3;
+    mainProgram = "${pname}";
+    maintainers = with lib.maintainers; [ _4r7if3x ];
+    platforms = lib.platforms.all;
```

Comment:
Heavily doubt it (not even Go works on all platforms). If it works on all platforms Go supports, then just leave it blank